util/log/file.go

```diff
@@ -349,7 +352,18 @@ func readAllEntriesFromFile(file FileInfo, startTimestamp, endTimestamp int64, m
 			}
 			return nil, false, err
 		}
-		if entry.Time >= startTimestamp && entry.Time <= endTimestamp {
+		var match bool
+		if pattern == nil {
+			match = true
+		} else {
+			args := []interface{}{}
+			for _, arg := range entry.Args {
+				args = append(args, arg.Str)
```

Comment:
I wish I could.  But I can see someone looking for a pattern that bridges the format string and the values.
"failure processing range range=1" or something similar